roofit/roofitcore/src/RooDataHist.cxx

```diff
@@ -1142,6 +1142,49 @@ RooPlot *RooDataHist::plotOn(RooPlot *frame, PlotOpt o) const
   return RooAbsData::plotOn(frame,o) ;
 }
 
+
```

Comment:
Can I ask to document new functions properly when they are introduced. It's never a better time than now, because you still know what is does. 🙂